js/apps/admin-ui/src/authentication/policies/OtpPolicy.tsx

```diff
@@ -238,6 +241,18 @@ export const OtpPolicy = ({ realm, realmUpdated }: OtpPolicyProps) => {
               labelOff={t("off")}
             />
           )}
+          <SwitchControl
+            name={"otpPolicyUseEmail"}
+            label={t("otpPolicyUseEmail")}
+            labelIcon={t("otpPolicyUseEmailHelp")}
+            labelOn={t("on")}
+            labelOff={t("off")}
+          />
```

Comment:
```suggestion
          <DefaultSwitchControl
            name={"otpPolicyUseEmail"}
            label={t("otpPolicyUseEmail")}
            labelIcon={t("otpPolicyUseEmailHelp")}
          />
```

Reply:
Didn't know that existed, I'll apply it :). Since there's an existing SwitchControl directly above it, should I adjust it there as well, or do I not touch the existing code because it's not the scope of this PR?

Reply:
I went ahead and turned the SwitchControl above also into DefaultSwitchControl.